Lib/fontTools/ttLib/tables/_h_e_a_d.py

```diff
@@ -65,6 +65,19 @@ def compile(self, ttFont):
 			if 'CFF ' in ttFont:
 				topDict = ttFont['CFF '].cff.topDictIndex[0]
 				self.xMin, self.yMin, self.xMax, self.yMax = intRect(topDict.FontBBox)
+			if 'CFF2' in ttFont:
```

Comment:
elif?

Reply:
Good point. Probably doesn’t make difference in practice, but good idea nevertheless.